app/src/main/java/com/android/halanvendordiscovery/data/details/mapper/MerchantToModel.kt

```diff
@@ -0,0 +1,15 @@
+package com.android.halanvendordiscovery.data.details.mapper
+
+import com.android.halanvendordiscovery.data.details.model.response.Merchant
+import com.android.halanvendordiscovery.domain.details.model.MerchantModel
+
+fun Merchant.toDomain() = MerchantModel(
```

Comment:
you can add a class mapper to convert from remote to domain Why do you use the extension function?

[How to abuse Kotlin extension functions](https://zone84.tech/programming/how-to-abuse-kotlin-extension-functions/)